src/tracker/detector/torchvision_detector.py

```diff
@@ -1,19 +1,18 @@
-import torchvision
-import torch
-
-
-import os
-from typing import List, Dict
+import ssl
+from typing import Dict, List
 
+import torch
+import torchvision
+from torchvision.models.detection import (
+    FasterRCNN_MobileNet_V3_Large_320_FPN_Weights,
+)
 
 from src.config.config import DetectorConfig
-from src.utils import resource_path
+from src.image.image import ImageObject
 from src.tracker.detector.detection import Detection
 from src.tracker.detector.detector import Detector
-from src.image.image import ImageObject
-from torchvision.models.detection import (
-    FasterRCNN_MobileNet_V3_Large_320_FPN_Weights,
-)
+
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
:grimacing: This line looks like you're messing around with the internals of a standard library. Put in a comment explaining what you're doing and why it's a good idea, because my first instinct is to never do this and run away from code that does.

Reply:
We would get an SSL Error
`URLError - <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)> - file_name='urllib/request.py' func_name='do_open' line_num=1351 resource rdk:service:vision/re_id model viam:vision:re-id-object-tracker`

Found the fix[ here](https://github.com/pytorch/pytorch/issues/33288)

Reply:
Great response! Put it in a comment in the code.

Reply:
done